dsgrid/cli/dsgrid.py

```diff
@@ -116,6 +121,7 @@ def cli(
         timer_stats_collector.enable()
     else:
         timer_stats_collector.disable()
+    ctx.params["offline"] = True
```

Comment:
Why keep this around?

Reply:
There is code elsewhere that depends on it. I tried to only comment-out code that we might eventually use again. It is conceivable that we have local and remote registries that can be synced. If you say, "no way", I can fully delete.